graphics/quirc/Portfile

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        dlbeer quirc 1.1 v
+revision            0
+categories-append   math
```

Comment:
This should be changed to: `categories graphics math`

Per lint errors:

```
port lint quirc: Error: Portfile parent directory graphics does not match primary category math
```

Reply:
Thanks done.